### ISSUE TITLE
Can't use `for` in `clerk/html` hiccup

### ISSUE THREAD

**Issue by jgoodhcg** [nextjournal/clerk#413]
```clojure
(clerk/html
 [:ul
  (for [x (range 1 4)]
    [:li x])])
 ```
    
Results in a blank screen and this error message in the browser console
    
 ```
 Error: :li is not ISeqable
 ```

**Comment by jgoodhcg** [comment]
Workaround right now is to include `hiccup` directly and call `hmtl` on the form to render a string and pass that into `clerk/html`.

```clojure
(require '[hiccup.core :refer [html]])

(clerk/html
  (html [:ul (for [x (range 1 4)] [:li x])]))
```

**Comment by mk** [comment]
Hi Justin, this is a duplicate of #395 which we fixed #398. I've tried this with the current `main` and it works:
<img width="350" alt="Screen Shot 2023-02-11 at 08 25 45" src="https://user-images.githubusercontent.com/1187/218246246-6414d59d-ea56-4d4a-bc95-eb3a5fb10e41.png">
